lib/routes.js

```diff
@@ -218,17 +218,23 @@ function addSiteController(router, site, providers) {
  * @param {Object} site
  * @returns {Promise}
  */
-function addSite(router, { providers, sessionStore }, site) {
+function addSite(router, { providers }, site) {
   site = _.defaults(site, { path: '/' }); // If path isn't explicitly set, set it.
   const path = site.path,
     siteRouter = express.Router();
 
   // authentication added first to protect all routes
   if (!_.isEmpty(providers)) {
-    auth.init(siteRouter, providers, site, sessionStore);
+    // Initialize auth module
+    amphoraAuth({
+      router: siteRouter,
+      providers,
+      site,
+      storage: db,
+      bus
```

Comment:
Is the auth module using the bus? I thought it was using the session store

Reply:
Are we saying the session store is now instantiated in the auth module and is no longer required to be passed in? I think that would be a be a breaking change in that we would be taking over the ability of the user to customize their session store. What do you think about passing in a reference to the session store here and then in amphora-auth you can instantiate it if a session store if one wasn't passed in.

Reply:
Yes, we are using the bus to publish the save and delete action of users, please see [here](https://github.com/clay/amphora-auth/pull/11/files#diff-71dfd080cffe4efa09fc54e1b1b92ef4R33) and [here](https://github.com/clay/amphora-auth/pull/11/files#diff-71dfd080cffe4efa09fc54e1b1b92ef4R48)

Reply:
Also, since we are dropping support for session store in Amphora v8 ([see here](https://github.com/clay/amphora/pull/635/files#diff-f5d8c0f33aa88aa6e119a0cdd5f1c40eR44) for warning message) I thought we were going to handle this through amphora-auth. But that's a better idea to avoid breaking changes

Reply:
Ah, I totally forgot about the bus for that. Good call. But yeah, we need to make sure not to drop support in v7. Nice job!